src/decorator/array/ArrayContains.ts

```diff
@@ -7,11 +7,11 @@ export const ARRAY_CONTAINS = "arrayContains";
  * Checks if array contains all values from the given array of values.
  * If null or undefined is given then this function returns false.
  */
-export function arrayContains(array: unknown, values: any[]) {
+export function arrayContains(array: unknown, values: any[]): boolean {
     if (!(array instanceof Array))
         return false;
 
-    return values.every(value => array.indexOf(value) !== -1);
+    return values.every(value => array.includes(value));
```

Comment:
@vlapo, is class-validator used on frontend? `includes` requires a polyfill in IE.

Reply:
the rest of the PR looks good.

Reply:
Yeah we should support browsers too. Unfortunately we do not have tests for this and also we do not have define minimal supported browser versions. 

Maybe we can use eslint plugin - https://github.com/amilajack/eslint-plugin-compat

Reply:
https://caniuse.com/#search=includes

Reply:
This is governed by the following eslint rule:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-includes.md

Reply:
Let's add https://github.com/amilajack/eslint-plugin-compat plugin and define config like `"browserslist": ["> 5%"],`

Also disable `prefer-includes` please.

Reply:
@rmuchall could you please work in this changes?

Reply:
Yes, apologies for the delay. I'll finish them off tomorrow :)

Reply:
Its okey. Just friendly reminder :)

Reply:
Done :)